mnefun/_mnefun.py

```diff
@@ -364,6 +367,11 @@ def do_processing(p, fetch_raw=False, do_score=False, push_raw=False,
              'Status',
              ]
     score_fun = p.score if p.score is not None else default_score
+    if len(inspect.getargspec(score_fun).args) == 2:
+        score_fun_two = score_fun
+
+        def score_fun(p, subjects, run_indices):
```

Comment:
@Eric89GXL does this also address the issue during scoring or is that on the user to resolve? I can see `run_indices` passed in but don't see it used. I am getting an I/O error during scoring for a subject with missing runs.